templates/head.pug

```diff
@@ -26,5 +26,3 @@ mixin head()
     <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
     <script src="https://oss.maxcdn.com/libs/respond.js/1.3.0/respond.min.js"></script>
   //[endif]
-  
-  include ../www/svg/coolhole-icons.svg
```

Comment:
i think i missed this convo, what happened here? including it in head didnt work?